tests/test_enable_scipts.py

```diff
@@ -0,0 +1,155 @@
+import komodo
+import os
+import json
+from komodo.data import Data
+
+
+def _create_script(kmd_prefix, kmd_pyver, kmd_release, target, template):
+    data = Data(extra_data_dirs=None)
+    os.mkdir(kmd_release)
+    with open("{}/{}".format(kmd_release, target), "w") as f:
+        f.write(
+            komodo.shell(
+                [
+                    "m4 {}".format(data.get("enable.m4")),
+                    "-D komodo_prefix={}".format(kmd_prefix),
+                    "-D komodo_pyver={}".format(kmd_pyver),
+                    "-D komodo_release={}".format(kmd_release),
+                    data.get(template),
+                ]
+            ).decode("utf-8")
+        )
+
+
+def _load_envs():
+    with open("pre_source.env") as pre:
+        pre_env = json.loads(pre.read())
+    with open("sourced.env") as sourced:
+        sourced_env = json.loads(sourced.read())
+    with open("post_disable.env") as post:
+        post_env = json.loads(post.read())
+
+    return pre_env, sourced_env, post_env
+
+
+TEST_SCRIPT_SIMPLE = """

```

Comment:
`"""\` with the `\` at the end to avoid an empty first line. ;)

Reply:
👍